sqlmesh/utils/pydantic.py

```diff
@@ -60,6 +60,16 @@ def _expression_encoder(e: exp.Expression) -> str:
     return e.meta.get("sql") or e.sql(dialect=e.meta.get("dialect"))
 
 
+def serialize_expressions(kwargs: t.Dict[str, t.Optional[exp.Expression]]) -> t.Dict[str, str]:
+    serialized_kwargs: t.Dict[str, str] = {}
+    for key, value in kwargs.items():
+        if isinstance(value, exp.Expression):
+            serialized_kwargs[key] = _expression_encoder(value)
+        else:
+            serialized_kwargs[key] = str(value)
```

Comment:
Checking whether `value` is an `Expression` seems redundant, based on `kwargs`' type hint.